frontend/encore/advanced-config.rst

```diff
@@ -38,6 +38,22 @@ But be careful not to accidentally override any config from Encore:
     // BAD - this replaces any extensions added by Encore
     // config.resolve.extensions = ['json'];
 
+If you absolutely need to overwrite or add to Encore config:
```

Comment:
i think `absolutely` is a bit strong as word, no?
what about `If you need to overwrite or add options that are not supported by Encore, use this config:`

and remove the sentence after your codeblock

Reply:
Thanks, I agree. I changed it.